mahmoodhamdi/selaty
# Add Profile View and Widgets to Home Feature

### Summary
This PR introduces the Profile View along with its associated widgets to the Home feature, enhancing the user interface and user experience of the profile section.

### What's Included
- **ProfileView Implementation**: 
  - A fully designed ProfileView with a clean and user-friendly layout.
  - Includes `CustomAppBar`, `ProfileInfo`, `ProfileGridView`, and `ProfileUtilBox` for a comprehensive profile display.
  
- **New Widgets**: 
  - `CustomBottomNavigationBar` and `CustomFloatingActionButton` for improved navigation within the app.
  - Modular widgets like `ProfileGridViewItem` and `ProfileUtilBoxItem` for enhanced code reusability and maintainability.

### Impact
- Improves the overall user interface of the profile section.
- Provides a solid foundation for further development of user-related features.

### How to Test
- Navigate to the ProfileView and ensure all elements are displayed correctly.
- Verify that the CustomBottomNavigationBar and CustomFloatingActionButton function as expected.

This update significantly enhances the profile section's design and usability, offering a more cohesive user experience.
